tests/test_sysfs.sh

```diff
@@ -0,0 +1,101 @@
+# Error on unset variables
+set -u
+
+# Load Linux version of _lp_battery()
+uname() { printf 'Linux'; }
+
+. ../liquidprompt --no-activate
+unset -f uname
+
+LP_ENABLE_BATT=1
+_LP_BATTERY_FUNCTION=__lp_battery_sysfs
+
+typeset -a battery_types battery_presents battery_status battery_capacities battery_out_statuses battery_values
+
+# Add test cases to these arrays like below
+
+# Empty ("") means file doesn't exist
+
+# Linux 5.11.0-38-generic #42~20.04.1-Ubuntu SMP Tue Sep 28 20:41:07 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
+battery_types+=("Battery")
+battery_presents+=("1")
+battery_statuses+=("Discharging")
+battery_capacities+=("67")
+battery_out_statuses+=(0)
+battery_values+=("67")
+battery_scopes+=("")
+
+# Full, not charging
+battery_types+=("Battery")
+battery_presents+=("")
+battery_statuses+=("")
+battery_capacities+=("")
+battery_out_statuses+=(4)
+battery_values+=("")
+battery_scopes+=("System")
+
+# Not a battery
+battery_types+=("Mains")
+battery_presents+=("")
+battery_statuses+=("")
+battery_capacities+=("")
+battery_out_statuses+=(4)
+battery_values+=("")
+battery_scopes+=("")
+
+# Wrong type of battery battery (Wireless mouse, some other)
+battery_types+=("Battery")
+battery_presents+=("")
+battery_statuses+=("")
+battery_capacities+=("")
```

Comment:
I'd like to see a value here in this case, to super verify that Liquidprompt doesn't pick it up.